app/[locale]/wallet/_components/transfer-block.tsx

```diff
@@ -10,15 +10,57 @@ import { Input } from '@/components/ui/input';
 import { Label } from '@/components/ui/label';
 import Row from '@/components/ui/row';
 import { Select, SelectContent, SelectGroup, SelectIcon, SelectItem, SelectTrigger, SelectValue } from '@/components/ui/select';
-import { useBallance } from '@/hooks/wallet/use-ballance';
+import Spinner from '@/components/ui/spinner';
+import { useAxonePayments } from '@/hooks/wallet/use-axone-payments';
 import { cn } from '@/lib/utils';
 
 const TransferBlock = () => {
   const [open, setOpen] = useState<boolean>(false);
-  const { balance, isFetchingBalance, balanceDenom } = useBallance();
+  const [amount, setAmount] = useState<number>(5);
```

Comment:
Please, use react-hook-form here and zod for validation.

Instead of alerts show errors below input